implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverMeterRegistry.java

```diff
@@ -140,10 +140,14 @@ protected void publish() {
             return;
         }
 
-        for (List<Meter> batch : MeterPartition.partition(this, config.batchSize())) {
+        // Stackdriver's API limits us to less than 200 events per call
+        int limit = 199;
```

Comment:
I think 200 is right. Perhaps before you had mostly Micrometer meters that translated to 1 Stackdriver `TimeSeries` each, and 1 meter that translated to 2 `TimeSeries` which put it over the limit when batching 200 meters?

Reply:
I've restored `200` in c99df29b4a30cfb318195241eef0f6e72d01e751 and I'll give it a second try on our systems and let you know.

Reply:
@shakuzen, I can confirm 200 is right :+1: